ROMFS/px4fmu_common/init.d/13013_deltaquad

```diff
@@ -107,12 +107,16 @@ then
 	param set SYS_COMPANION 57600
 fi
 
-set MIXER vtol_quad_x
+set MIXER deltaquad
 set PWM_OUT 1234
 set PWM_RATE 400
 
-set MIXER_AUX vtol_delta
-set PWM_ACHDIS 3
+param set PWM_MAIN5_DIS 1500
+param set PWM_MAIN6_DIS 1500
+param set PWM_MAIN7_DIS 900
+param set PWM_MAIN8_DIS 900
```

Comment:
I have vehicles with clients, they need this forced. when i confirm they have upgraded i will move it to AUTOCNF